### PR TITLE
Rename "Case Study Excerpt" block to "Showcase"

### PR DESCRIPTION
**Pull request by yahilmadakiya**
This PR will adds :point_down: 

1. Rename "Case Study Excerpt" block to "Showcase".
2. Use Middleware to devlop block.

### CODE REVIEW

File: blocks/showcase/index.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	},
+
+	getEditWrapperProps() {
+		return { 'data-align': 'wide' };
+	},
+
+	edit( props, middleware ) {
+		const {
+			attributes: {
+				showCaseLink,
```
Comment (sayedtaqui) [inline blocks/showcase/index.js]: Please fix trailing spaces.

File: blocks/showcase/index.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+				<div className="image-container">
+					{ middleware.fields.showCaseImage }
+				</div>
+
+				<div className="info-container">
+					{ middleware.inspectorControls }
+					{ middleware.fields.showCaseTitle }
+					{ middleware.fields.showCaseContent }
+					{ middleware.fields.showCaseLink }
+					{ showCaseLink ? <a href={ showCaseLink } title="__( 'Read More' )" className="button secondary">{ __( 'Read More' ) }</a> : '' }
```
Comment (sayedtaqui) [inline blocks/showcase/index.js]: `"__( 'Read More' )"` is a string not JS

File: blocks/showcase/index.js (hunk 1 above)
Comment (yahilmadakiya) [reply blocks/showcase/index.js]: Removed.

File: blocks/showcase/index.js (hunk 2 above)
Comment (yahilmadakiya) [reply blocks/showcase/index.js]: Fixed.

File: blocks/showcase/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+				imagePlaceholder: true,
+				removeButtonText: __( 'Remove' ),
+			},
+		},
+
+		showCaseLink: {
+			type: 'string',
+			field: {
+				type: 'link',
+				placement: __( 'inspector' ),
```
Comment (sayedtaqui) [inline blocks/showcase/index.js]: This isn't translated string

File: blocks/showcase/index.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+				</div>
+			);
+		}
+
+		return (
+			<div className={ className + ' showcase-wrapper alignwide' }>
+				{ imageContent }
+				<div className="info-container">
+					{ showCaseLink ? ( <a href={ showCaseLink } ><h3 className="showcase-title">{ showCaseTitle ? showCaseTitle : '' }</h3></a> ) : ( <h3 className="showcase-title">{ showCaseTitle ? showCaseTitle : '' }</h3> ) }
+					<div className="showcase-content">{ showCaseContent ? showCaseContent : '' }</div>
```
Comment (sayedtaqui) [inline blocks/showcase/index.js]: this condition isn't required `showCaseContent ? showCaseContent : ''`

**Comment by sayedtaqui** [comment]
Rest are looking good 👍

**Pull request merged**